js/mainjs/addIPCListeners.js

```diff
@@ -30,7 +34,10 @@ module.exports = function(config, mainWindow) {
 			default:
 				console.error('Unknown dialog ipc');
 		}
-		event.returnValue = response ? response : null;
+		// TODO: Make async and adapt plugins
```

Comment:
can you explain this todo a bit better? I'm guessing there's not going to be enough time to do this.